app/main.py

```diff
@@ -1 +1,27 @@
-# write your code here
+from decimal import Decimal
+
+import json
+
+
+def calculate_profit(file_name):
+    with open(file_name, "r") as trade_report:
+        operations = json.load(trade_report)
+
+    earned_money = 0
+    matecoin_account = 0
+
+    for operation in operations:
+        matecoin_price = Decimal(operation["matecoin_price"])
+
+        if operation["sold"]:
+            earned_money += Decimal(operation["sold"] * matecoin_price)
+            matecoin_account -= Decimal(operation["sold"])
+
+        elif operation["bought"]:
```

Comment:
It would be better to use if here instead of elif. Because theoretically there can be both 'sold' and 'bought'.

Reply:
Yes, it is better to use `if` here.

Reply:
Fixed